lib/glue_scripts/transformation-sql/mlw_ichis_expansion.sql

```diff
@@ -0,0 +1,39 @@
+select	day(start_date)     AS day,
+		month(start_date)   AS month,
+		year(start_date)    AS year,
+		start_date date,
+		end_date,
+		deviceid,
+		Testtype,
+		Participant_ID_Number,
+		Name_of_participant,
+		Training_ID_Number,
+		District_Health_Zone,
+		District,
+		Reporting_health_facility_name,
+		Position,
+		Qualification,
+		Age_category,
+		Reporting_Catchment_Area_Name,
+		Catchment_area_population_size,
+		Years_worked_as_a_health_worker,
+		Gender,
+		Q1_Score,
+		Q3_score,
+		Q4_Score,
+		Q5_Score,
+		Q6_Score,
+		Q8_Score,
+		Q10_Score,
+		Q7_Score,
+		Q8_Score2,
+		Q9_score,
+		Q10_Score3,
+		score,
+		Final_score,
+		Status
+from mlw_ichis_expansion_data.mlw_ichis_expansion

```

Comment:
don't forget to change this to Malawi yah Bwighane.

Reply:
@bwighane, I am wondering; how many commits do you have for this PR? I am asking because I also see two other commits in addition to the "dynamic job" job commit.

Reply:
2 commits. The first was for the SQL files the the etl job script and step functions